src/Symfony/Component/Intl/Intl.php

```diff
@@ -64,6 +66,11 @@ final class Intl
      */
     const REGION_DIR = 'regions';
 
+    /**
+     * The directory name of the zone data.
+     */
+    const ZONE_DIR = 'zones';
```

Comment:
do we want to track new dirs here? tend to inline it in `Timezones::getPath()` instead, and deprecate these const in general.